ietf/doc/views_doc.py

```diff
@@ -977,14 +975,8 @@ def document_pdfized(request, name, rev=None, ext=None):
 
     try:
         pdf = doc.pdfized()
-    except URLFetchingError:
-        # retry once, then give up
-        try:
-            pdf = doc.pdfized()
-        except URLFetchingError as exception:
-            return render(request, "doc/weasyprint_failed.html",
-                          dict(error=f'{type(exception).__name__}: {exception}'),
-                          status=504)
+    except:
```

Comment:
Should probably be `except Exception` - a bare `except` catches low-level non-`Exception` interruptions that shouldn't normally be caught without re-raising.